packages/clerk-js/src/ui/elements/UserPreview.tsx

```diff
@@ -7,6 +7,11 @@ import { descriptors, Flex, Text, useLocalizations } from '../customizables';
 import type { PropsOfComponent, ThemableCssProp } from '../styledSystem';
 import { UserAvatar } from './UserAvatar';
 
+// FIXME Make this accept an interface with the superset of fields in:
```

Comment:
Is this a todo?

Reply:
Yes, this component had become a bit of a mess